internal/commands/show_status.go

```diff
@@ -167,6 +179,52 @@ func createStatusAttachment(ctx context.Context, client bot.Client, logger log.L
 	return attach, nil
 }
 
+func treatMessage(ctx context.Context, client bot.Client, logger log.Logger, msg string) (string, error) {
+	msg = treatHere(msg)
+	msg, err := treatUsersMentions(ctx, client, logger, msg)
+	if err != nil {
+		return "", err
+	}
+
+	return msg, nil
+}
+
+func treatHere(msg string) string {
+	x := []string{
+		"here",
+		"channel",
```

Comment:
This will works for @here and @channel, and its ok. 

What about the user groups (i.e. @hydra-team) for example? We should continue to notify  user groups?

Reply:
To fix a mention to a user group, we will need to get all user groups and find the correct one. So, we will fix this in a future PR:

https://github.com/ResultadosDigitais/hellper/issues/126